src/components/navigation/Navigation.tsx

```diff
@@ -48,28 +51,43 @@ const Navigation = ({ onNavigationToggle, open, width }: Props) => {
                 <List aria-label="main application navigation">
                     <ListItemLink
                         icon={<ExploreIcon />}
-                        title="Dashboard"
-                        link="/dashboard"
-                        key="Dashboard"
+                        title={intl.formatMessage({
```

Comment:
Given the repetition, I am curious if it would be possible to pass `routeDetails.focalRoute` to the `ListItemLink` component and have component set `title`, `link`, and `key` properties internally.

Reply:
I think so - I just didn't want to take the time to refactor this stuff. But it's a good place for something like that.